scripts/update_deps.py

```diff
@@ -214,20 +240,23 @@ def find_latest_jax_nightly() -> Optional[tuple[str, str, str]]:
     or None if no build is found.
   """
 
-  # Fetch JAX project page
+  # Fetch JAX project page.
   jax_links = fetch_pep503_page(_JAX_PROJECT_URL)
   if not jax_links:
-    logger.error(f'Could not fetch JAX packages from {_JAX_PROJECT_URL}')
+    logger.error("Could not fetch JAX packages from %s", _JAX_PROJECT_URL)
```

Comment:
Curious about the reason for avoiding the f-string.

Reply:
Ah, it's just that `ruff` throw me this lint warning, and I think Yaosh at one point gave a nit on some other PR that loggers should avoid f-strings for performance reasons. I can change all of this back though.

Reply:
I'll just change back to f-string for now to be consistent with the rest of this file.